services/src/handlers/projects.rs

```diff
@@ -35,6 +38,44 @@ where
         );
 }
 
+mod error {
```

Comment:
There is the Snafu-Option `module(error)` to do exactly this.

Reply:
80f2649acc5e1021ec9aa1130515e646d5e6b5d1